content/agent/_index.md

```diff
@@ -23,15 +23,15 @@ Agent v6 is available. <a href="https://docs.datadoghq.com/agent/faq/upgrade-to-
 
 ## What is the Agent?
 
-The Datadog Agent is a piece of software that runs on your hosts. Its job is to faithfully collect events and metrics and bring them to Datadog on your behalf so that you can do something useful with your monitoring and performance data. The Datadog Agent is open source: view the source code on GitHub for [Agent v6][2].
+The Datadog Agent is a software that runs on your hosts. It collects events and metrics from hosts and sends them to Datadog. From where you can analyze your monitoring and performance data. The Datadog Agent is open source and its source code is available on GitHub at [DataDog/datadog-agent][2].
```

Comment:
From history it was visible that the link to the GitHub repo was recently changed to version 6. Having a direct link to the repo without mentioning version will save us from remembering to update it with every new version.